src/main/java/frc/robot/Robot.java

```diff
@@ -39,25 +39,28 @@ public void robotInit() {
 		inst = NetworkTableInstance.create();
 
 		// start a NT4 client
-		inst.startClient4("robot network table client");
+		inst.startClient4(Constants.NETWORK_TABLE_CLIENT);
 
 		// connect to a roboRIO with team number TEAM
-		inst.setServerTeam(8248);
+		inst.setServerTeam(Constants.TEAM_NUMBER);
 
 		// connect to a specific host/port
-		inst.setServer("10.0.0.2", NetworkTableInstance.kDefaultPort4);
-		USB2 = inst.getTable("photonvision").getSubTable("USB2.0_PC_CAMERA");
-		if (OPEN_TAG_WINDOW) {
+		inst.setServer(Constants.NETWORK_TABLES_SERVER, NetworkTableInstance.kDefaultPort4);
+		USB2 = inst.getTable(Constants.PhotonVisionConstants.PHOTON_NETWORK_TABLES_NAME).getSubTable(Constants.PhotonVisionConstants.PHOTON_CAMERA_NAME);
+		//
+    if (Constants.PhotonVisionConstants.OPEN_TAG_WINDOW) {
```

Comment:
Not sure if it's because I'm on mobile but the indentation looks all messed up - can you confirm that the style is the same as the existing code?